lib/seahorn/HornCex.cc

```diff
@@ -373,5 +386,62 @@ namespace seahorn
     out.keep ();
   }
 
+  static void sliceErrorTrace(Function &F, BmcTrace &trace) {
```

Comment:
I am not sure I see the difference in semantics. I think at this point, there is no abstraction done by the BMC trace. How does the difference manifest itself in the code? If the change is very small, we might merge it with `reduceToRegion`  and use a flag to separate between the two semantics.